app/src/assets/i18n/en.json

```
@@ -70,7 +70,8 @@
             "COMPANY_DETAILS": "Company details",
             "NAME": "Name",
             "ADDRESS": "Address",
-            "POST_CODE": "Post code"
+            "POST_CODE": "Post code",
```

should be a single word
```suggestion
            "POSTCODE": "Postcode",
```

---

changed